compiler/job.go

```diff
@@ -146,14 +147,26 @@ func (j *Job) Parallelize(n int) error {
 	return j.optimizer.Parallelize(n)
 }
 
-func Parse(src string, filenames ...string) (ast.Op, error) {
+func ParseNoWrap(src string, filenames ...string) (ast.Op, error) {
 	parsed, err := parser.ParseZed(filenames, src)
 	if err != nil {
 		return nil, err
 	}
 	return ast.UnpackMapAsOp(parsed)
 }
 
+func Parse(src string, filenames ...string) (ast.Op, error) {
+	op, err := ParseNoWrap(src, filenames...)
+	return wrapScope(op), err
+}
+
+func wrapScope(op ast.Op) ast.Op {
+	if seq, ok := op.(*ast.Sequential); ok {
+		op = &ast.Scope{Kind: "Scope", Body: seq}
+	}
+	return op
+}
+
```

Comment:
This is just for this PR.  Going away in the `seq` branch.  I hacked things up so `ast.Scope` can be the entry but this will change to `ast.Seq` in the next bit of work.  This is what I meant in the PR comment about this being clunky.  It is temporary.

Reply:
I guess my point is that the diffs in this PR and the the seq branch PR will be smaller if you deal with this in parser.peg, that's just a nit.